tests/Unit/Evolution/Systems/GeneralizedHarmonic/BoundaryConditions/Test_BjorhusImpl.cpp

```diff
@@ -0,0 +1,890 @@
+                      8.413256058790063e+19, 8.413256071059287e+19,
+                      7.122948320807626e+19, 7.639071432523556e+19,
+                      8.413256071059287e+19, 9.445502273748306e+19}};
+        not_initialized = false;
+      }
+      if (lic0[i] == 299. and lic1[i] == 0.5 and lic2[i] == 0.5) {
+        spec_vals = {{7.122948296615527e+19, 7.122948312744913e+19,
+                      7.122948308712188e+19, 7.122948320807332e+19,
+                      7.122948312744913e+19, 7.639071424305924e+19,
+                      7.639071420196061e+19, 7.639071432523039e+19,
+                      7.122948308712188e+19, 7.639071420196061e+19,
+                      8.413256058789768e+19, 8.413256071058824e+19,
+                      7.122948320807332e+19, 7.639071432523039e+19,
+                      8.413256071058824e+19, 9.445502273747671e+19}};
+        not_initialized = false;
+      }
+      if (lic0[i] == 299.5 and lic1[i] == 0.5 and lic2[i] == 0.5) {
+        spec_vals = {{7.122948296615504e+19, 7.122948312745003e+19,
+                      7.122948308712251e+19, 7.122948320807479e+19,
+                      7.122948312745003e+19, 7.639071424306128e+19,
+                      7.639071420196236e+19, 7.639071432523298e+19,
+                      7.122948308712251e+19, 7.639071420196236e+19,
+                      8.413256058789916e+19, 8.413256071059056e+19,
+                      7.122948320807479e+19, 7.639071432523298e+19,
+                      8.413256071059056e+19, 9.445502273747989e+19}};
+        not_initialized = false;
+      }
+      if (lic0[i] == 300. and lic1[i] == 0.5 and lic2[i] == 0.5) {
+        spec_vals = {{7.122948296615481e+19, 7.122948312745094e+19,
+                      7.122948308712312e+19, 7.122948320807626e+19,
+                      7.122948312745094e+19, 7.639071424306332e+19,
+                      7.639071420196412e+19, 7.639071432523558e+19,
+                      7.122948308712312e+19, 7.639071420196412e+19,
+                      8.413256058790063e+19, 8.413256071059287e+19,
+                      7.122948320807626e+19, 7.639071432523558e+19,
+                      8.413256071059287e+19, 9.445502273748306e+19}};
+        not_initialized = false;
+      }
+      if (not_initialized) {
+        ASSERT(true,
+               "Not checking the correct face, coordinates not recognized");
+      }
+      for (size_t a = 0; a <= VolumeDim; ++a) {
+        for (size_t b = a; b <= VolumeDim; ++b) {
+          spec_bc_dt_v_minus.get(a, b)[i] =
+              gsl::at(spec_vals, a * (1 + VolumeDim) + b);
+        }
+      }
+    }
+
+    // Compare values returned by BC action vs those from SpEC
+    CHECK_ITERABLE_APPROX(local_bc_dt_v_minus, spec_bc_dt_v_minus);
+  }
+}
+}  // namespace
+
+SPECTRE_TEST_CASE("Unit.Evolution.Systems.GeneralizedHarmonic.BCBjorhus.VMinus",
+                  "[Unit][Evolution]") {
+  pypp::SetupLocalPythonEnvironment local_python_env{""};
```

Comment:
Nope, because `Bjorhus.py` imports several modules from `PointwiseFunctions` which is accessible from root only. So this environment has to be created from root.